components/log-viewer-webui/client/src/pages/IngestPage/Jobs/index.tsx

```diff
@@ -81,13 +38,43 @@ interface JobsProps {
  * @return
  */
 const Jobs = ({className}: JobsProps) => {
+    const {refreshInterval} = useRefreshIntervalStore();
+    const [jobs, setJobs] = useState<JobData[]>([]);
+    const intervalIdRef = useRef<number>(0);
+
+    const update = useCallback(() => {
```

Comment:
can we make the callback an `async` one? since we don't really show any UI for any Error, letting it throw is better than catching the locally thrown error then simply `console.error`.
```
    const fetchJobsStats = useCallback(async () => {
        const beginTimestamp = dayjs().subtract(DAYS_TO_SHOW, "days")
            .unix();
        const {data: resp} = await querySql<QueryJobsResp>(getQueryJobsSql(beginTimestamp));
        const newJobs = resp
            .map((item): JobData => convertQueryJobsItemToJobData(item));

        setJobs(newJobs);
    }, []);
```